.github/workflows/test.yml

```diff
@@ -3,6 +3,29 @@ name: Tests
 on: [push, pull_request]
 
 jobs:
+
+  benchmarks:
+    name: 📈 Benchmarks
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - run: pipx install poetry
```

Comment:
Shouldn't this come after the setup-python?

Reply:
Good point, I honestly copied this job over from strawberry, where' it's been running smoothly for a while. I can change the order if you prefer.

Reply:
I'll try with the other order later. But will merge as is for now.